### ISSUE TITLE
Wiki update: no instant run in AS v. 3.5 or higher

### ISSUE THREAD

**Issue by gszczesz** [ftctechnh/ftc_app#752]
Wiki page: https://github.com/ftctechnh/ftc_app/wiki/Disabling-Android-Studio-Instant-Run
Please note that Android Studio 3.5 and above no longer supports instant run because it is incompatible with the Jack toolchain, and as such has been removed.

**Comment by FROGbots-4634** [comment]
This is true, however v3.5 introduces a new replacement for Instant Run, called Apply Changes. https://developer.android.com/studio/releases#3-5-0

**Comment by NoahAndrews** [comment]
Apply Changes doesn't change the APK that gets deployed though, unlike Instant Run.
